src/diffusers/models/transformer_2d.py

```diff
@@ -93,6 +94,10 @@ def __init__(
         norm_type: str = "layer_norm",
         norm_elementwise_affine: bool = True,
         attention_type: str = "default",
+        caption_channels: int = None,
+        output_type: str = "vanilla_dit",
+        interpolation_scale: int = 1,
+        use_additional_conditions=False,
```

Comment:
Can this be True for some checkpoints? If this is always the same param, let's not add it here